apps/client/src/pages/LocationPage.js

```diff
@@ -67,9 +79,9 @@ const LocationPage = ({ topic }) => {
         <LocationFinder
           setAddress={setAddress}
           setErrorMessage={setErrorMessage}
-          postalCode={context.address?.postalCode}
-          houseNumberFull={context.address?.houseNumberFull}
-          houseNumber={context.address?.houseNumberFull}
+          postalCode={context.address?.[slug]?.postalCode}
```

Comment:
I like to see that the data is from the context. 
But i think to best practice is too destructure everything. What do you think?

Reply:
Fixed. Can you review?